### PR TITLE
Update Xcodeproj minimum requirement

### PR DESCRIPTION
**Pull request by dnkoutso**
This is required for 1.8.0.beta.1 to function see bug fix release: https://github.com/CocoaPods/Xcodeproj/releases/tag/1.11.1

### CODE REVIEW

**Comment by CocoaPodsBarista** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please include a CHANGELOG entry to credit yourself! <br />
You can find it at <a href="https://github.com/CocoaPods/CocoaPods/blob/master/CHANGELOG.md">CHANGELOG.md</a>.</td>
    </tr>
  </tbody>
</table>

Here's an example of your CHANGELOG entry:

```markdown
* Update Xcodeproj minimum requirement  
  [dnkoutso](https://github.com/dnkoutso)
  [#issue_number](https://github.com/CocoaPods/CocoaPods/issues/issue_number)
```

*note*: There are two invisible spaces after the entry's text.


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: .gitignore [hunk 1]
```diff
@@ -44,7 +44,3 @@ coverage/
 
 # Bundler files
 /.bundle
-
-# trunk source
-spec/fixtures/spec-repos/trunk/**
```
Comment (dnkoutso) [inline .gitignore]: @igor-makarov is this right?

File: spec/fixtures/spec-repos/trunk/CocoaPods-version.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (dnkoutso) [inline spec/fixtures/spec-repos/trunk/CocoaPods-version.yml]: @igor-makarov shoud this be checked in?

File: spec/fixtures/spec-repos/trunk/CocoaPods-version.yml.etag [hunk 3]
```diff
@@ -0,0 +1 @@
+"38ceffa2727d3e9df40f72da77311837-ssl"
```
Comment (dnkoutso) [inline spec/fixtures/spec-repos/trunk/CocoaPods-version.yml.etag]: @igor-makarov last question should the `.etag` files be checked in? I dont think we want them.

Comment (dnkoutso) [reply spec/fixtures/spec-repos/trunk/CocoaPods-version.yml.etag]: Decided to delete them

Comment (igor-makarov) [reply spec/fixtures/spec-repos/trunk/CocoaPods-version.yml.etag]: Makes sense, they aren't used unless podspec is being updated.

File: spec/fixtures/spec-repos/trunk/CocoaPods-version.yml (hunk 2 above)
Comment (igor-makarov) [reply spec/fixtures/spec-repos/trunk/CocoaPods-version.yml]: Yeah. It's definitely in use.

Comment (dnkoutso) [reply spec/fixtures/spec-repos/trunk/CocoaPods-version.yml]: do we need to check it in though? Would it change each time a new version is released and have to be committed?

Comment (dnkoutso) [reply spec/fixtures/spec-repos/trunk/CocoaPods-version.yml]: I guess what I am asking is if we should let tests download it each time and add it to `.gitignore`

Comment (igor-makarov) [reply spec/fixtures/spec-repos/trunk/CocoaPods-version.yml]: I guess that works too. I dunno, this bit isn't really exact science 😂

**Pull request merged**